.travis.yml

```diff
@@ -21,7 +21,7 @@ before_deploy:
 - ./.travis/install_install4j
 - ENGINE_VERSION=$(grep "engine_version" game_engine.properties | sed 's/.*= *//g')
 - TAGGED_VERSION="$ENGINE_VERSION.$TRAVIS_BUILD_NUMBER"
-- ./.travis/build_installers $TAGGED_VERSION
+- TAGGED_VERSION=$TAGGED_VERSION ./gradlew release
```

Comment:
Hmm, I don't think they are required.  It's working for me both locally (bash) and on [Travis](https://travis-ci.org/ssoloff/triplea-game-triplea/builds/251834683#L4438) (dash).

I can't find a Bourne shell reference to give you to prove this, but I tried a little experiment in my bash shell:

```shell
$ node -e "console.log(process.env['FOO'])"
undefined
$ FOO=bar node -e "console.log(process.env['FOO'])"
bar
$ FOO=bar baz node -e "console.log(process.env['FOO'])"
bash: baz: command not found...
$ FOO="bar baz" node -e "console.log(process.env['FOO'])"
bar baz
```

It looks like the shell stops parsing the environment variable value when it encounters whitespace.  So, technically, we could surround `$TAGGED_VERSION` with quotes just in case it contains whitespace, but it shouldn't be necessary for the command itself:

```shell
TAGGED_VERSION="$TAGGED_VERSION" ./gradlew release
```

And please note that this is **not** overwriting the existing `TAGGED_VERSION` environment variable in the parent shell.  It is simply defining a new variable (with the same name) in the child process.  For example:

```shell
$ echo $FOO

$ FOO=bar
$ echo $FOO
bar
$ FOO=baz node -e "console.log(process.env['FOO'])"
baz
$ echo $FOO
bar
```